vector/src/main/java/im/vector/app/features/settings/devices/v2/VectorSettingsDevicesFragment.kt

```diff
@@ -181,13 +181,18 @@ class VectorSettingsDevicesFragment :
         handleLoadingStatus(state.isLoading)
     }
 
-    private fun renderSecurityRecommendations(inactiveSessionsCount: Int, unverifiedSessionsCount: Int) {
+    private fun renderSecurityRecommendations(
+            inactiveSessionsCount: Int,
+            unverifiedSessionsCount: Int,
+            isCurrentSessionVerified: Boolean,
+    ) {
         if (unverifiedSessionsCount == 0 && inactiveSessionsCount == 0) {
             hideSecurityRecommendations()
         } else {
             views.deviceListHeaderSectionSecurityRecommendations.isVisible = true
             views.deviceListSecurityRecommendationsDivider.isVisible = true
-            views.deviceListUnverifiedSessionsRecommendation.isVisible = unverifiedSessionsCount > 0
+
+            views.deviceListUnverifiedSessionsRecommendation.isVisible = unverifiedSessionsCount > 0 && isCurrentSessionVerified
```

Comment:
Could be nice to start using Paparazzi to test the different state of the View.
There is an example in `PaparazziExampleScreenshotTest` and a doc `screenshot_testing.md` is the project.

Reply:
Yes, we will plan it. For now, the screens are still subject to change. Once the feature is stable, we will add some tests.